eng/Versions.props

```
@@ -12,7 +12,7 @@
   <PropertyGroup>
     <VersionPrefix>6.0.100</VersionPrefix>
     <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
-    <PreReleaseVersionIteration>7</PreReleaseVersionIteration>
+    <PreReleaseVersionIteration>8</PreReleaseVersionIteration>
```

According to WillG, this is how we mark the release for rc1.
```suggestion
    <PreReleaseVersionIteration>1</PreReleaseVersionIteration>
    <PreReleaseBrandingLabel>rc</PreReleaseBrandingLabel>
```